h/static/styles/developer-page.scss

```diff
@@ -0,0 +1,8 @@
+// The developer-page block implements the styles used by the "Developer" page
+// in the account/profile pages.
+
+.developer-page__api-token-input {
```

Comment:
Something that isn't clearly defined in our docs at the moment is when something should be a block vs an element.
The basic principle is that anything which is reusable on its own should be a block.

In the frontend code this is usually straightforward because the same applies to a component (or Angular directive), so often component == block.

In the website templates this is less obvious. Here I would suggest that `api-token-input` should be its own block.